apps/overlay/src/pages/Markets/marketsTable.js

```diff
@@ -17,6 +17,7 @@ import {
 import useSDK from "../../hooks/useSDK";
 import { useMultichainContext } from "../../state/multichain/useMultichainContext";
 import { useAccount } from "../../hooks/useAccount";
+import { toWei } from "overlay-sdk/dist/common/utils/formatWei";
```

Comment:
In application it should be imported toWei function the same way? Maybe it is possible to export it in constructor(sdk)?